doc/version_history.rst

```diff
@@ -2,6 +2,11 @@
 Version History
 ===============
 
+v0.6.5
```

Comment:
I think these are quite substantial changes for a patch release. Consider making this v0.7.0.... Patch releases are mostly for backward compatible bug fixes.

Reply:
Done in the commit: [da7203f](https://github.com/lsst-ts/ts_m2/pull/33/commits/da7203f14d4c12b1939ad5b81a5373a29e69023b). Thanks!